app/controllers/admin/articles_controller.rb

```diff
@@ -89,6 +89,8 @@ def article_params
                                     :published_at, :tags, :categories,
                                     :image, :image_description, :css, :hide_layout,
                                     :header_background_color, :header_text_color,
+									                  :short_path,
+                                    contributions_attributes: [
```

Comment:
`contribution_attributes` is listed twice now, and nested incorrectly. I'm guessing this is the result of a bad merge conflict.